includes/admin/payments/class-payments-table.php

```diff
@@ -656,6 +656,7 @@ public function get_bulk_actions() {
 
 		if ( 'trash' === $this->get_status() ) {
 			$action['restore'] = __( 'Restore', 'easy-digital-downloads' );
+			$action['delete']  = __( 'Delete Permanently', 'easy-digital-downloads' );
 		} else {
 			$action['trash'] = __( 'Trash', 'easy-digital-downloads' );
```

Comment:
Let's change this to "Move to Trash" to match core.

Reply:
@spencerfinnell  this is on the filtered view showing only items which are already in the trash. This is making the trashed orders table bulk actions equivalent to what is in Core (screenshots in the related issue).

Reply:
@robincornett Sorry, the comment wasn't for the addition (line 659) but the unchanged line (now 661).

Core Posts:
<img width="217" alt="Screen Shot 2020-05-20 at 11 51 50 AM" src="https://user-images.githubusercontent.com/491124/82468098-6d6c3e80-9a90-11ea-879d-6986e91905e8.png">

EDD Orders:
<img width="211" alt="Screen Shot 2020-05-20 at 11 52 04 AM" src="https://user-images.githubusercontent.com/491124/82468129-75c47980-9a90-11ea-8c51-85b1442326aa.png">

Reply:
My apologies, I misunderstood. Yes, definitely a good call to change that as well. Thanks!